ocaml/xapi/monitor_types.ml

```diff
@@ -15,8 +15,7 @@
  * @group Performance Monitoring
 *)
 
-open Stdext
-open Xstringext
+module Strext = Stdext.Xstringext.String
```

Comment:
Here you use it, but you may as well ditch it and use `Astring`, as we did in few other modules already (with the intention of killing `Stdext.Xstringext` once we were able to)